### ISSUE TITLE
ADD FastFlow

### ISSUE THREAD

**Issue by kyakuno** [axinc-ai/ailia-models#824]
An unofficial pytorch implementation of [FastFlow: Unsupervised Anomaly Detection and Localization via 2D Normalizing Flows](https://arxiv.org/abs/2111.07677) (Jiawei Yu et al.).
https://github.com/gathierry/FastFlow

**Comment by kyakuno** [comment]
Apache2

**Comment by miyamotost** [comment]
pretrained modelがない

**Comment by kyakuno** [comment]
学習も必要なAnormalyDetectionモデルであり、推論のみをサポートするailia SDKでは実装できない。
